index.html

```diff
@@ -151,6 +151,16 @@ <h2>Data Model</h2>
             property, and <code>JsonSchemaCredential2023</code> for usage of a [[JSON-Schema]] represented as a
             <a>verifiable credential</a>.
          </p>
+         <p>
+            Implementers may find use in packaging a JSON Schema as a verifiable credential when they wish to
+            leverage features of the [[VC-DATA-MODEL-2]], answering questions such as:
+            <ul>
+               <li>Who is the author of this schema? (provided by the <code>issuer</code> property)</li>
+               <li>Why should the author be trusted for providing this schema? (provided by the <code>evidence</code> property)</li>
+               <li>Is it schema still in use? (provided by the <code>validFrom</code>, <code>validUntil</code>, and <code>credentialStatus</code> properties)</li>
+               <li>Has the schema been tampered with? (provided by [[VC-DATA-INTEGRITY]] or [[VC-JWT]])</li>
```

Comment:
Better to refer to the vc specs dir directly...

Reply:
here? https://w3c.github.io/vc-specs-dir/#proof

I don't see vc-cose-jose

Reply:
because this has not been merged: https://github.com/w3c/vc-specs-dir/pull/14

Reply:
it would be nice to have a single 'securing mechanism' section but I understand why it's not like that

as an alternative could wait until https://github.com/w3c/vc-json-schema/pull/191 is merged and then link here to the new content integrity protection section